### PR TITLE
change ansible_ + etcd_network_iface ipv4 to ansible_default_ipv4

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ etcd_data_dir: /var/lib/etcd
 
 etcd_master_group_name: etcd-master
 
-etcd_secure: True
+etcd_secure: False
```
Comment (andrewrothstein) [inline defaults/main.yml:8]: I'd rather users have to think about/decide/understand what it takes to secure their cluster and opt out of securing it if they feel that's the right solution for them.

File: vars/main.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-etcd_public_address: "{% if etcd_use_ips %}{{hostvars[inventory_hostname]['ansible_' + etcd_network_iface]['ipv4']['address']}}{% else %}{{inventory_hostname}}{% endif %}"
+etcd_public_address: "{% if etcd_use_ips %}{{hostvars[inventory_hostname]['ansible_default_ipv4']['address']}}{% else %}{{inventory_hostname}}{% endif %}"
 etcd_scheme: "{% if etcd_secure %}https{% else %}http{% endif %}"
-etcd_cluster: "{% for host in groups[etcd_master_group_name] %}{{hostvars[host]['inventory_hostname']}}={{etcd_scheme}}://{% if etcd_use_ips %}{{hostvars[host]['ansible_' + etcd_network_iface]['ipv4']['address']}}{% else %}{{host}}{% endif %}:2380{% if not loop.last %},{% endif %}{% endfor %}"
+etcd_cluster: "{% for host in groups[etcd_master_group_name] %}{{hostvars[host]['inventory_hostname']}}={{etcd_scheme}}://{% if etcd_use_ips %}{{hostvars[host]['ansible_default_ipv4']['address']}}{% else %}{{host}}{% endif %}:2380{% if not loop.last %},{% endif %}{% endfor %}"
```
Comment (andrewrothstein) [inline vars/main.yml:4]: I'd like to maintain the option to specify/control the network interface explicitly.

**Comment by patsevanton** [comment]
Please review PR

**Review by andrewrothstein — CHANGES_REQUESTED** [review]

**Comment by andrewrothstein** [comment]
Take a look at v1.2.0 that I just published to the galaxy. It will use the default ipv4 network interface by default now.

**Pull request closed without merging**